specs/dotNet4.5.1_should_pass.ps1

```diff
@@ -3,6 +3,9 @@ Framework "4.5.1x86"
 task default -depends MsBuild
 
 task MsBuild {
-  $output = &msbuild /version 2>&1
-  Assert ($output -NotLike "12.0") '$output should contain 12.0'
+    if ( $IsOSX -OR $IsLinux ) {}
+    else {
+        $output = &msbuild /version 2>&1
```

Comment:
Yeah, I actually just commented out the build tests but after doing some research it looks like the better way is to download the dotnet cli and then use `dotnet build` to get the information. I will make those changes and also fix the logo.